### PR TITLE
refactor: removing opentracing module and references

### PR DESCRIPTION
**Pull request by elbiocaetano**

### CODE REVIEW

**Review by christopher-cudennec — COMMENTED** [review]
Looks good; more changes than expected though. Was a good exercise to find remaining code snippets that still needed to be changed! 😃

**Review by christopher-cudennec — APPROVED** [review]

File: sda-commons-client-jersey/src/main/java/org/sdase/commons/client/jersey/filter/ContainerRequestContextHolder.java [hunk 1]
```diff
@@ -85,22 +80,20 @@ public static <V> Callable<V> transferRequestContext(Callable<V> callable) {
...
-    return new TracedCallable<>(
-        () -> {
-          ContainerRequestContextHolder containerRequestContextHolder =
-              new ContainerRequestContextHolder();
-          try {
-            if (contextMap != null) {
-              MDC.setContextMap(contextMap);
-            }
-            containerRequestContextHolder.filter(requestContext);
+    return () -> {
```
Comment (JoergSiebahn) [inline sda-commons-client-jersey/src/main/java/org/sdase/commons/client/jersey/filter/ContainerRequestContextHolder.java:83]: Shouldn't this be wrapped with `Context.wrap` (or similar) now?

File: sda-commons-client-jersey/src/main/java/org/sdase/commons/client/jersey/filter/ContainerRequestContextHolder.java [hunk 2]
```diff
@@ -55,23 +52,21 @@ public static Runnable transferRequestContext(Runnable runnable) {
...
-    return new TracedRunnable(
-        () -> {
-          ContainerRequestContextHolder containerRequestContextHolder =
-              new ContainerRequestContextHolder();
-          try {
-            if (contextMap != null) {
-              MDC.setContextMap(contextMap);
-            }
-            containerRequestContextHolder.filter(requestContext);
+    return () -> {
```
Comment (JoergSiebahn) [inline sda-commons-client-jersey/src/main/java/org/sdase/commons/client/jersey/filter/ContainerRequestContextHolder.java:55]: Shouldn't this be wrapped with `Context.wrap` (or similar) now?

Comment (elbiocaetano) [reply sda-commons-client-jersey/src/main/java/org/sdase/commons/client/jersey/filter/ContainerRequestContextHolder.java:55]: Fixed on https://github.com/SDA-SE/sda-dropwizard-commons/pull/2040

**Pull request merged**